packages/js/manifests/polywrap/src/formats/polywrap.plugin/0.2.0.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface PluginManifest {
+  /**
+   * Polywrap manifest format version.
+   */
+  format: "0.2.0";
+  /**
+   * Basic project properties.
+   */
+  project: {
+    /**
+     * Name of this project.
+     */
+    name: string;
+    /**
+     * Type of this project.
+     */
+    type: string;
+  };
+  /**
+   * Project source files.
+   */
+  source: {
+    /**
+     * Path to the project's entry point.
+     */
+    module?: string;
+    /**
+     * Path to the project's graphql schema.
+     */
+    schema: string;
+  };
+  /**
+   * Project extension files (build, meta, deploy, infra).
+   */
+  extensions?: {
+    /**
+     * Path to the project build manifest file.
+     */
+    build?: string;
+    /**
+     * Path to project metadata manifest file.
+     */
+    meta?: string;
+    /**
+     * Path to project deploy manifest file.
+     */
+    deploy?: string;
+    /**
+     * Path to project infra manifest file.
+     */
+    infra?: string;
+  };
+  /**
+   * Redirects for the schema's imports.
+   */
+  import_redirects?: {
```

Comment:
I think it'd actually make a lot of sense to move the import_redirects within the `source` property. `source.import_redirects`. Thoughts?

Reply:
Since `import_redirects` realistically only serves the schema, I'd agree with this. I'll make the changes!